tests/playwright-test/to-have-screenshot.spec.ts

```diff
@@ -405,6 +405,8 @@ test('should compile with different option combinations', async ({ runTSC }) =>
             threshold: 0.2,
             maxDiffPixels: 10,
             maxDiffPixelRatio: 0.2,
+            //maxDiffSize: 2,
```

Comment:
We'll need an actual test for this feature that demonstrates (at least on one platform that we support) where the different size is a legit use case.